data-model/fdo-types/0.3.0/digital-media-objects/schema/digital-entity.json

```diff
@@ -0,0 +1,182 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/fdo-types/0.3.0/digital-media-objects/digital-entity.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "DigitalObject Version 0.3.0",
+  "type": "object",
+  "properties": {
+    "@id": {
+      "type": "string",
+      "description": "The unique identifier of the object",
+      "pattern": "https:\\/\\/doi\\.org\\/10\\.\\d+\\/[-\\w]+",
+      "examples": [
+        "https://doi.org/10.22/XXX-XXX-XXX"
+      ]
+    },
+    "@type": {
+      "type": "string",
+      "description": "The type of the object",
+      "const": "ods:DigitalEntity"
+    },
+    "ods:ID": {
+      "type": "string",
+      "description": "The unique digital identifier of the object",
+      "pattern": "https:\\/\\/doi\\.org\\/10\\.\\d+\\/[-\\w]+",
+      "examples": [
+        "https://hdl.handle.net.org/20.5000.1025/XXX-XXX-XXX"
+      ]
+    },
+    "ods:version": {
+      "type": "integer",
+      "description": "The version of the object, each change generates a new version",
+      "minimum": 0,
+      "examples": [
+        1
+      ]
+    },
+    "ods:created": {
+      "type": "string",
+      "description": "The timestamp that the object version was created in DiSSCo",
+      "format": "date-time",
+      "examples": [
+      ]
+    },
+    "ods:type": {
+      "type": "string",
+      "description": "The DOI or handle to the FDO type of the object",
```

Comment:
regex doesn't cover handle

Reply:
Removed handle from this as it should be a DOI right